lumapps/api/client.py

```diff
@@ -112,6 +101,7 @@ def get_user_api(self, email: str, prune: bool = True) -> "LumAppsClient":
             token_getter=self.get_token_getter(email),
             prune=prune,
             api_info=self.api_info,
+            auth_info=self._auth_info,
```

Comment:
you need the auth info if you have the token ?
is it not either one or another ?